.github/workflows/synthetics-stack-e2e-init.yml

```diff
@@ -0,0 +1,36 @@
+---
+#
+# This workflow sets the status check `synthetics-stack-e2e` instantly to 'failure'
+# the synthetics-stack-e2e.yml is triggered with a PR comment and is responsible for setting it to success or failure.
```

Comment:
Will this also run for external contributors? IIUC, we only wanted to run this particular GitHub check for any changes caused by any Elastician. Can you confirm?

Reply:
IIUC, only the trigger comment should do something if it's an elastic member.

We still want to mark it as untested.

Reply:
IIUC, this comment should never run on PRs coming from external contributors. Hence PRs from external contributors will always have the failed github check and will never be changed.

IMO, that's not ideal, it can confuse people whether the PR is ready to be merged because the check failed for that reason or another one.